.kokoro/completeness-check.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
It looks like you might also need to make the file permissions executable.

Reply:
this file is alr an executable.